plugins/generator-1.16.5/forge-1.16.5/templates/villagertrade.java.ftl

```diff
@@ -37,10 +37,24 @@ import net.minecraft.entity.merchant.villager.VillagerTrades;
 
 @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE) public class ${name}Trade {
 
+    @SubscribeEvent public static void registerWanderingTrades(WandererTradesEvent event) {
```

Comment:
This should only be called if there are wandering trades present, same in 1.18.x

Reply:
It's not possible to check that unless to get actual entry to check the profession type

Reply:
Please check marker assign function and how to do logic inside freemarker

Reply:
I already did!

Reply:
The mechanic you need here is used in other templates, so surely you can do that here too

Reply:
Ok it's fixed